src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -57,8 +63,62 @@ export function HighRiskBreachLayout(props: HighRiskBreachLayoutProps) {
   // The non-null assertion here should be safe since we already did this check
   // in `./[type]/page.tsx`:
   const { title, illustration, content, exposedData, type } = pageData!;
-  const hasBreaches = type !== "none";
+  const isHighRiskBreachesStep = type !== "none";
   const isStepDone = type === "done";
+  const hasExposedData = exposedData.length > 0;
+
+  // TODO: Write unit tests MNTOR-2560
+  /* c8 ignore start */
```

Comment:
Are we just ignoring this code from coverage stats until the unit tests are written?

Reply:
Yes! We’ll still need to add unit tests for the resolution interaction and were thinking about generally adding a good pattern to mock network requests for unit tests.